### DISCUSSION TITLE
No se ven propiedades hijo y padre en el admin

### DISCUSSION THREAD

**Discussion by seiturra** [Cotalker/documentation#381] [4. Bug Reports] [answered]
En la sección de propiedades se ven las secciones de padres e hijos pero no se muestran las propiedades que tiene esta propiedad como hija ni los hijos de esta.

**Comment by jdurandpaz** [comment]
![image](https://user-images.githubusercontent.com/50208672/132217597-1ae29d83-385a-4522-bc47-c3951e8fa2c8.png)

**Comment by agallardol** [comment, marked as answer]
@SeIturra Muchas gracias por reportar, esto ya se encuentra corregido y disponible en Cotalker
